content/javascript/concepts/strings/terms/charCodeAt/charCodeAt.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: '.charCodeAt()'
+Description: 'Returns an integer representing the UTF-16 character code at the specified index'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent: 
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **charCodeAt()** method used with javascript strings returns an positive integer representing UTF-16 character code at the given index.
+
+## Syntax
+
+```pseudo
+sentence.charCodeAt()
+```
+
+The `.charCodeAt()` method returns a positive integer for the provided variable `sentence`.
```

Comment:
Instead of this line, describe the above two parameters in the following format:
- `string`: describe this parameter
- `index`: describe this parameter